### ISSUE TITLE
Provider "azurestack" v0.7.0 is not compatible with Terraform 0.12.2.

### ISSUE THREAD

**Issue by wellsluo** [terraform-providers/terraform-provider-azurestack#91]
Run Terraform 0.12.2 with Azure Stack configuration file.  

Error reported from "Terraform init":

Provider "azurestack" v0.7.0 is not compatible with Terraform 0.12.2.

Provider version 0.6.0 is the latest compatible version. Select it with
the following constraint:

    version = "~> 0.6"

Terraform checked all of the plugin versions matching the given constraint:
    ~> 0.6

Consult the documentation for this provider for more information on
compatibility between provider and Terraform versions.

Alternatively, upgrade to the latest version of Terraform for compatibility with newer provider releases.

So question is what is the Terraform version we need to use for Azure Stack provider 0.7?

**Comment by bottkars** [comment]
0.7 works with < 0.12, latest 0.11 works

**Comment by tombuildsstuff** [comment]
@wellsluo @bottkars apologies about this, this was a missed setting on the release; I'll open an issue internally to get this resolved

**Comment by bottkars** [comment]
Great . If you have have a beta I could Grab meanwhile it would be great. Was not able to build tiny myself 😎

Von meinem iPhone gesendet

Am 18.06.2019 um 07:58 schrieb Tom Harvey <notifications@github.com<mailto:notifications@github.com>>:


@wellsluo<https://github.com/wellsluo> @bottkars<https://github.com/bottkars> apologies about this, this was a missed setting on the release; I'll open an issue internally to get this resolved

**Comment by tombuildsstuff** [comment]
hey @wellsluo @bottkars 

Just to let you know that this has been resolved and that `terraform init` now returns 0.7.0 using Terraform 0.12:

```
$ terraform init

Initializing the backend...

Initializing provider plugins...
- Checking for available provider plugins...
- Downloading plugin for provider "azurestack" (terraform-providers/azurestack) 0.7.0...

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary
```

I'm going to close this issue for the moment but please let us know if that doesn't work for you and we'll take another look :)

Thanks!
